src/helpers/current-loans.json

```diff
@@ -8,7 +8,8 @@
       "annualised_return": "8.60",
       "term_remaining": "1648243234000",
       "ltv": "48.80",
-      "amount": "85754"
+      "amount": "85754",
+      "invested": false
```

Comment:
don't put "invested" field here. You can just have a separate state in your component with "invested" flag